src/pages/WebComponents.tsx

```diff
@@ -277,32 +281,30 @@ export default function WebComponents() {
             ></tds-icon>
           </tds-button>
         </div>
-        <tds-popover-canvas
-          placement="auto"
-          selector="#trigger"
-          className="tds-u-p2"
-        >
-          <h2>A popover canvas!</h2>
-          <p>Where you can put anything you want!</p>
-          <p>
-            <a
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://digitaldesign.scania.com"
-            >
-              Even links!
-            </a>
-          </p>
+        <tds-popover-canvas placement="auto" selector="#trigger">
+          <div className="tds-u-p2">
+            <h2>A popover canvas!</h2>
+            <p>Where you can put anything you want!</p>
+            <p>
+              <a
```

Comment:
The link text is pretty hard to read in dark mode when it has been pressed/visited, is this something we want to address?
![image](https://github.com/scania-digital-design-system/tegel-react-demo/assets/48509966/c50f2d06-6748-4c0b-be79-83ac47e655d2)